src/main/java/br/edu/ufcg/computacao/alumni/api/http/request/Match.java

```diff
@@ -91,7 +93,13 @@
                 throw new InvalidParameterException(Messages.PAGE_MUST_BE_AN_INTEGER);
             }
 
-            Page<PendingMatch> pendingMatches = ApplicationFacade.getInstance().getAlumniPendingMatches(token, p);
+            try {
+                if (minScore != null) Integer.parseInt(minScore);
```

Comment:
Code would be cleaner if convertion of minScore from String to int would be made at this point. The signature of the facade method would be:
ApplicationFacade.getInstance().getAlumniPendingMatches(String token, int p, int minScore).